src/Resources/ResourceManager/Implementation/CmdletBase/DeploymentStacksCreateCmdletBase.cs

```diff
@@ -79,6 +85,8 @@ protected DeploymentStacksCreateCmdletBase()
             Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Parameter file to use for the template.")]
         [Parameter(ParameterSetName = ParameterFileTemplateSpecParameterSetName,
             Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Parameter file to use for the template.")]
+        [Parameter(ParameterSetName = ByParameterFileWithNoTemplateParameterSetName,
+            Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Parameter file to use for the deployment.")]
```

Comment:
Only help message in the first parameter set will be considered in help MDs. Please change to a conclusive HelpMessage.

Reply:
Done!